modules/cpmso-ts-mhc-etcd-degraded.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cpmso-troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="cpmso-ts-etcd-degraded_{context}"]
+= Recovering a degraded etcd Operator
+
+Certain situations can cause the etcd Operator to become degraded. 
+
+//This was originally written as solely the outcome of an MHC mishap, but iiuc that is merely one situation that could cause this.
+For example, while performing remediation, the machine health check might delete a control plane machine that is hosting etcd. If the etcd member is not reachable at that time, the etcd Operator becomes degraded. 
+
+When the etcd Operator is degraded, manual intervention is required to force the Operator to remove the failed member and restore the cluster state.
+
+.Procedure
+
+. List the control plane machines in your cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get machines -l machine.openshift.io/cluster-api-machine-role==master -n openshift-machine-api -o wide
+----
++
+A failed control plane machine has the `STATE` value `stopped`.
+
+. Edit the machine CR for the failed control plane machine by running the following command:
++
+[source,terminal]
+----
+$ oc edit machine <control_plane_machine_name>
+----
+
+. Remove the contents of the `lifecycleHooks` parameter from the failed control plane machine and save your changes.
```

Comment:
We probably want to add a note here saying that you should make sure there are two healthy Machines before doing this, if you do this to more than one machine at a time you'll risk etcd losing quorum and then data loss

Reply:
Ok cool, I will pull some text about recovery in here